contributing/documentation/documentation-map/README.md

```diff
@@ -193,6 +193,7 @@ Here you can also find what articles are incomplete and missing.
         - <span class="missing">[Authentication](../missing-and-outdated-list/index.html#Authentication)</span>
         - Commands
         - Events
+    - Layout Embedded
```

Comment:
Should this be changed to `Embedded Layout` as of [page's title](https://github.com/RocketChat/docs/pull/681/files#diff-21a9468c9abb601c7f3650003dd3ae19R1)